cmd/cmd.go

```diff
@@ -59,6 +59,7 @@ func (c *BotContext) VerifyRequest(r *http.Request, body []byte) error {
 	return nil
 }
 
+//nolint:govet
```

Comment:
After reading a bit about the ecosystem, I realized we're using golangci-lint which has [false positives](https://golangci-lint.run/usage/false-positives). I've added a few `//nolint`, but I really don't know what I'm doing here...

Reply:
we are missing godocs I guess; we can add them after this PR is merged. 

godocs usually have a space between the slashes and the first word; if your file is gofmt'ed that should do it